pymemcache/test/test_client.py

```diff
@@ -1255,6 +1255,36 @@ class MyClient(Client):
         assert isinstance(client.client_pool.get(), MyClient)
 
 
+class TestPooledClientIdleTimeout(ClientTestMixin, unittest.TestCase):
+    def make_client(self, mock_socket_values, **kwargs):
+        mock_client = Client(None, **kwargs)
+        mock_client.sock = MockSocket(list(mock_socket_values))
+        client = PooledClient(None, pool_idle_timeout=60, **kwargs)
+        client.client_pool = pool.ObjectPool(lambda: mock_client)
+        return client
+
+    def test_free_idle(self):
+        num_removed = 0
+
+        def increment_removed(obj):
+            nonlocal num_removed
```

Comment:
`nonlocal` isn't supported under Python 2.x.

Reply:
Okay, changed to use a `Counter` object instead.